presto-native-execution/presto_cpp/main/common/Counters.h

```diff
@@ -114,6 +114,13 @@ constexpr folly::StringPiece kCounterMappedMemoryRawAllocBytesSizeClass{
 // largest SizeClass cannot accommodate, are counted towards this counter.
 constexpr folly::StringPiece kCounterMappedMemoryRawAllocBytesLarge{
     "presto_cpp.mapped_memory_raw_alloc_bytes_large"};
+/// Number of bytes currently queued in PrestoExchangeSource waiting for
```

Comment:
Comments probably need to change after moving the counter away from ExchangeSource